src/LanguageServer/Protocol/Features/AbstractAnalyzerAssemblyLoaderProvider.cs

```diff
@@ -14,21 +18,50 @@ namespace Microsoft.CodeAnalysis.Host;
 /// </summary>
 internal abstract class AbstractAnalyzerAssemblyLoaderProvider : IAnalyzerAssemblyLoaderProvider
 {
-    private readonly Lazy<IAnalyzerAssemblyLoader> _shadowCopyLoader;
+    private readonly Lazy<IAnalyzerAssemblyLoader> _sharedShadowCopyLoader;
+    private readonly ImmutableArray<IAnalyzerAssemblyResolver> _externalResolvers;
 
     public AbstractAnalyzerAssemblyLoaderProvider(ImmutableArray<IAnalyzerAssemblyResolver> externalResolvers)
     {
         // We use a lazy here in case creating the loader requires MEF imports in the derived constructor.
-        _shadowCopyLoader = new Lazy<IAnalyzerAssemblyLoader>(() => CreateShadowCopyLoader(externalResolvers));
+        _sharedShadowCopyLoader = new Lazy<IAnalyzerAssemblyLoader>(() => CreateShadowCopyLoader(
+#if NET
```

Comment:
nit: could avoid this #if directive if CreateShadowCopyLoader had it's param default to null

Reply:
done.